docs/Design-and-Modeling/Configure-Spectral.md

```diff
@@ -0,0 +1,13 @@
+# Configure Spectral Rules 
+
+![Configure Spectral](../assets/images/spectral-config.png)
+
+## What 
+Spectral, Stoplight’s linting tool, can be configured to create custom rulesets to match your organization's style guides. Enable or disable rules, create new rules, change their severity, or add new descriptions.
+
+## How 
+1. Select a **project** 
+2. Click **+** and select **File** from the dropdown 
+3. Input ```.spectral.yml``` for the file name 
+4. Select the newly created ```.spectral.yml``` file 
+5. Customize the ruleset via **Write** View 
```

Comment:
Add link to the bottom here to learn more about rulesets, kind of like what's shown in the read panel in studio for spectral ruleset.

Reply:
done